app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -2,21 +2,39 @@ const fields = {
   metadataKey: {
     id: 'metadata-column-header',
     name: 'metadata-column-header',
-    validation: [ {
-      validator: isNotEmpty,
-      message: 'Enter a column header'
-    } ]
+    validation: [
+      {
+        validator: isNotEmpty,
+        message: 'Enter a column header'
+      },
+      {
+        validator: isValidLength,
+        message: 'The column header is an invalid length'
```

Comment:
These message should ideally follow the guidance in https://design-system.service.gov.uk/components/error-message/

Text input docs might help here https://design-system.service.gov.uk/components/text-input/

In this case we'd probably be going for 
>The column header must be 50 characters or less

Reply:
Thanks - I have updated it.